src/main/java/no/difi/sdp/client2/internal/EbmsForsendelseContainer.java

```diff
@@ -0,0 +1,21 @@
+package no.difi.sdp.client2.internal;
+
+import no.digipost.api.representations.EbmsForsendelse;
+
+public class EbmsForsendelseContainer {
```

Comment:
Det er ikke noe i veien for å la denne ta inn kun bytes igjen og kalle det `SomethingSomethingMedFakturerbareBytes`, men da føles klassen bare som et hack for å få med mer data.